qcore/im.py

```diff
@@ -153,23 +156,27 @@ def get_unit(self):
         elif self.name in [IMEnum.PGV, IMEnum.AI]:
             return "cm/s"
         elif self.name in [IMEnum.CAV, IMEnum.FAS]:
-            return "gs"  # FAS could also be cm/s depending on calculation / implementation
+            return (
+                "gs"  # FAS could also be cm/s depending on calculation / implementation
+            )
         elif self.name in [IMEnum.Ds575, IMEnum.Ds595, IMEnum.Ds2080]:
             return "s"
-        elif self.name in [IMEnum.MMI, IMEnum.IESDR]:
+        elif self.name in [IMEnum.MMI]:
             return ""  # MMI is dimensionless & Ratios are dimensionless
```

Comment:
Update this comment too. As there are no longer any ratios here.

Reply:
Done. Just removed "Ratios are dimensionless" bit.